giantswarm-plugin.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     - name: TC_KUBECONFIG
     - name: CP_KUBECONFIG
     - name: CLUSTER_ID
+    - name: E2E_PROVIDER
```

Comment:
bikeshedding as its finest, but should we call it just `PROVIDER` ?

Reply:
Works for me.